cypress/pages/clusters.po.ts

```diff
@@ -259,6 +259,17 @@ export class ClustersPage {
     cy.get('div').should(Condition.Contain, 'No clusters available.');
   }
 
+  static verifyNoExternalClusters(): void {
+    if (Mocks.enabled()) {
+      cy.intercept({method: RequestType.GET, path: Endpoint.Clusters}, []);
```

Comment:
You are intercepting the internal cluster requests here, for external clusters use `Endpoint.ExternalClusters`.